client/components/RelatedProducts/Outfit/OutfitProductCard.jsx

```diff
@@ -46,8 +42,16 @@ const useStyles = makeStyles({
     top: '5px',
     right: '5px',
     color: '#442C2E'
+  },
+  sale: {
+    marginLeft: '5px',
+    color: red[400],
+  },
+  strikethrough: {
+    textDecoration: 'line-through',
```

Comment:
I like the consistency with the overview styles! I'd be happy to change my styles if there is anything you think could look better about the sale prices.